src/token/token.service.ts

```diff
@@ -149,8 +169,19 @@ export class TokenService {
       });
   }
 
+  async getStringifiedToken(token: string, clientIp: string): Promise<any> {
+    if (
+      this.configService.get<string>('app.env') === 'development' &&
```

Comment:
change logic to not equal to 'production' so that this can be enabled for all non-prod environments if needed